### PR TITLE
Replace persons with users

### PR DESCRIPTION
**Pull request by dalju005-xamk**

### CODE REVIEW

File: src/components/providers/authentication-provider.tsx [hunk 1]
```diff
@@ -80,8 +81,8 @@ const AuthenticationProvider = ({ children }: Props) => {
    * Sets the logged in timebank person from keycloak ID into global state
    */
   const getPersonsList = async () => {
-    const fetchedPersons = await personsApi.listPersons({ active: true });
-    setPersons(fetchedPersons);
+    const fetchedPersons = await usersApi.listUsers({ active: true });
```
Comment (Iiro127) [inline src/components/providers/authentication-provider.tsx:84]: I dont think "active: true" is needed. (?)

**Review by Iiro127 — CHANGES_REQUESTED** [review]
Also I think renaming would be in place, for example "const fetchedPersons" to "const fetchedUsers" and such.

File: src/components/providers/authentication-provider.tsx (hunk 1 above)
Comment (dalju005-xamk) [reply src/components/providers/authentication-provider.tsx:84]: I agree. Made changes, also named const to fetchedUsers

Also fixed other issue in
src/components/providers/authentication-provider.tsx
line 291 added draft.false to remove problem there

**Review by Iiro127 — APPROVED** [review]

**Review by AfoninOleksandr — COMMENTED** [review]
Mostly fine 
Rename new lists and careful about deleting personsAPI related code, comment out for now

File: src/components/providers/authentication-provider.tsx
[the remark this answers is not in the document]
Comment (dalju005-xamk) [reply src/components/providers/authentication-provider.tsx:1]: Done

**Review by AfoninOleksandr — APPROVED** [review]
Looks good for now

File: src/atoms/person.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import type { DailyEntry, Person, PersonTotalTime } from "../generated/client";
 import type { DailyEntryWithIndexSignature, PersonWithTotalTime } from "../types";
 import type { UsersAvatars } from "src/generated/homeLambdasClient";
 
+
```
Comment (Jdallos) [inline src/atoms/person.ts:6]:
```suggestion
```

Comment (Jdallos) [reply src/atoms/person.ts:6]: No need for double line break here (or anywhere)

File: src/components/screens/vacation-requests-screen.tsx [hunk 3]
```diff
@@ -287,7 +287,8 @@ const VacationRequestsScreen = () => {
           message: vacationData.message,
           createdAt: new Date(),
           updatedAt: new Date(),
-          days: vacationData.days
+          days: vacationData.days,
+          draft: false
```
Comment (Jdallos) [inline src/components/screens/vacation-requests-screen.tsx:291]: Why was this change added, is it essential?

**Review by Jdallos — CHANGES_REQUESTED** [review]

File: src/components/screens/vacation-requests-screen.tsx (hunk 3 above)
Comment (dalju005-xamk) [reply src/components/screens/vacation-requests-screen.tsx:291]: Remove the change and made it back to way it was

File: src/components/screens/vacation-requests-screen.tsx [hunk 4]
```diff
@@ -287,7 +287,9 @@ const VacationRequestsScreen = () => {
           message: vacationData.message,
           createdAt: new Date(),
           updatedAt: new Date(),
-          days: vacationData.days
+          days: vacationData.days,
+          draft: false,
+
```
Comment (Jdallos) [inline src/components/screens/vacation-requests-screen.tsx:292]:
```suggestion
```

Comment (Jdallos) [reply src/components/screens/vacation-requests-screen.tsx:292]: Remove unnecessary line break

**Review by Jdallos — APPROVED** [review]
